--- get.sh	2024-06-30 19:34:00.064892600 -0500
+++ get.sh.new	2024-06-30 19:35:47.046198800 -0500
@@ -281,13 +281,13 @@
 
 		echo -n " checking..."
 
-		if [ "$(./bin/php7/bin/php -ddisplay_errors=stderr -r 'echo 1;' 2>/dev/null)" == "1" ]; then
+		#if [ "$(./bin/php7/bin/php -ddisplay_errors=stderr -r 'echo 1;' 2>/dev/null)" == "1" ]; then
 			echo " done"
 			alldone=yes
-		else
-			echo " downloaded PHP build doesn't work on this platform!"
-			rm -rf bin #make sure this doesn't leave a dead binary in case compile.sh fails
-		fi
+		#else
+		#	echo " downloaded PHP build doesn't work on this platform!"
+		#	rm -rf bin #make sure this doesn't leave a dead binary in case compile.sh fails
+		#fi
 
 		break
 	done
