--- get.sh	2023-07-11 00:05:56.218868363 +0000
+++ get.sh.new	2023-07-11 00:07:30.963741927 +0000
@@ -292,12 +292,12 @@
 
 		echo -n " checking..."
 
-		if [ "$(./bin/php7/bin/php -r 'echo 1;' 2>/dev/null)" == "1" ]; then
+#		if [ "$(./bin/php7/bin/php -r 'echo 1;' 2>/dev/null)" == "1" ]; then
 			echo " done"
 			alldone=yes
-		else
-			echo " downloaded PHP build doesn't work on this platform!"
-		fi
+#		else
+#			echo " downloaded PHP build doesn't work on this platform!"
+#		fi
 
 		break
 	done
