--- pm_install.sh.old	2025-07-11 20:50:13.781476700 -0500
+++ pm_install.sh	2025-07-11 20:50:27.778989400 -0500
@@ -294,13 +294,13 @@
 		fi
 		echo -n " checking..."
 
-		if [ "$("$php_path/php" -ddisplay_errors=stderr -r 'echo 1;' 2>/dev/null)" == "1" ]; then
+		#if [ "$("$php_path/php" -ddisplay_errors=stderr -r 'echo 1;' 2>/dev/null)" == "1" ]; then
 			echo " done"
 			alldone=yes
-		else
-			echo " downloaded PHP build doesn't work on this platform!"
-			rm -rf bin #make sure this doesn't leave a dead binary in case compile.sh fails
-		fi
+		#else
+		#	echo " downloaded PHP build doesn't work on this platform!"
+		#	rm -rf bin #make sure this doesn't leave a dead binary in case compile.sh fails
+		#fi
 
 		break
 	done
